pkg/app/config.go

```diff
@@ -22,13 +22,16 @@ package app
 
 import (
 	"fmt"
+	logging "github.com/op/go-logging"
+	yaml "gopkg.in/yaml.v2"
 	"io/ioutil"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I find it weird to have this imported into the config. Seems that the thing that uses this should be in a util package or part of the clients package directly. @shawn-hurley @eriknelson thoughts?

Reply:
This would be the getSecretData method so I'm fine with moving that out to a util package.